<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>129</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ecuaciones Cuadraticas</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>45</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>Raices</string>
   </property>
   <property name="buddy">
    <cstring>raices_text</cstring>
   </property>
  </widget>
  <widget class="QTextEdit" name="raices_text">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="a_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>X^2 +</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="b_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>X +</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="c_edit"/>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>100</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>80</width>
     <height>96</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QPushButton" name="raices">
    <property name="text">
     <string>Raices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graficar">
    <property name="text">
     <string>Graficar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="limpiar">
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="salir_btn">
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>a_edit</tabstop>
  <tabstop>b_edit</tabstop>
  <tabstop>c_edit</tabstop>
  <tabstop>raices</tabstop>
  <tabstop>graficar</tabstop>
  <tabstop>limpiar</tabstop>
  <tabstop>salir_btn</tabstop>
  <tabstop>raices_text</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>salir_btn</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>129</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
